draft-irtf-cfrg-hpke.md

```diff
@@ -1205,6 +1205,23 @@ interface; see {{hpke-export}} for more details.
 
 # API Considerations {#api-considerations}
 
+## Auxiliary Application Information
+
+HPKE has two places at which applications can specify auxiliary authenticated information:
+(1) during context construction via the Setup `info` parameter, and (2) during Context
+operations, i.e., with the `aad` parameter for `Open()` and `Seal()`, and the `exporter_context` parameter
+for `Export()`. Application information applicable to multiple operations on a single Context
+should use the Setup `info` parameter. This avoids redundantly processing this information for
+each Context operation. In contrast, application information that varies on a per-message basis
+should be specified via the Context APIs (`Seal()`, `Open()`, or `Export()`).
+
+Applications that only use the single-shot APIs described in {{single-shot-apis}} should use the
+Setup `info` parameter for specifying auxiliary authenticated information. Implementations which
+only expose single-shot APIs should not allow applications to use both Setup and Context auxiliary
```

Comment:
I would refer to these as `info` and `aad` rather than Setup and Context.  Or "Setup `info`" and "Context `aad`".

Reply:
Done.